src/assets/css/dropdown-effects/fade-left.css

```diff
@@ -0,0 +1,115 @@
+  -webkit-transform-origin: center center;
+  -moz-transform-origin: center center;
+  -ms-transform-origin: center center;
+  -o-transform-origin: center center;
+  transform-origin: center center;
+  -webkit-transform: translateX(40px) skewX(7deg);
+  -moz-transform: translateX(40px) skewX(7deg);
+  -ms-transform: translateX(40px) skewX(7deg);
+  -o-transform: translateX(40px) skewX(7deg);
+  transform: translateX(40px) skewX(7deg);
+  -o-transition: -o-transform 0.3s, opacity 0.3s;
+  -ms-transition: -ms-transform 0.3s, opacity 0.3s;
+  -moz-transition: -moz-transform 0.3s, opacity 0.3s;
+  -webkit-transition: -webkit-transform 0.3s, opacity 0.3s;
+}
+
+.wsmenu>.wsmenu-list>li>ul.sub-menu>li:hover>ul.sub-menu {
+  opacity: 1;
+  visibility: visible;
+  -webkit-transform: rotateX(0deg);
+  -moz-transform: rotateX(0deg);
+  -ms-transform: rotateX(0deg);
+  -o-transform: rotateX(0deg);
+  transform: rotateX(0deg);
+}
+
+.wsmenu>.wsmenu-list>li>ul.sub-menu>li>ul.sub-menu>li>ul.sub-menu {
+  opacity: 0;
+  visibility: hidden;
+  -webkit-transform-origin: center center;
+  -moz-transform-origin: center center;
+  -ms-transform-origin: center center;
+  -o-transform-origin: center center;
+  transform-origin: center center;
+  -webkit-transform: translateX(40px) skewX(7deg);
+  -moz-transform: translateX(40px) skewX(7deg);
+  -ms-transform: translateX(40px) skewX(7deg);
+  -o-transform: translateX(40px) skewX(7deg);
+  transform: translateX(40px) skewX(7deg);
+  -o-transition: -o-transform 0.3s, opacity 0.3s;
+  -ms-transition: -ms-transform 0.3s, opacity 0.3s;
+  -moz-transition: -moz-transform 0.3s, opacity 0.3s;
+  -webkit-transition: -webkit-transform 0.3s, opacity 0.3s;
+}
+
+.wsmenu>.wsmenu-list>li>ul.sub-menu>li>ul.sub-menu>li:hover>ul.sub-menu {
+  opacity: 1;
+  visibility: visible;
+  -webkit-transform: rotateX(0deg);
+  -moz-transform: rotateX(0deg);
+  -ms-transform: rotateX(0deg);
+  -o-transform: rotateX(0deg);
+  transform: rotateX(0deg);
+}
+
+.wsmenu>.wsmenu-list>li>.wsmegamenu {
+  opacity: 0;
+  visibility: hidden;
+  -webkit-transform-origin: center center;
+  -moz-transform-origin: center center;
+  -ms-transform-origin: center center;
+  -o-transform-origin: center center;
+  transform-origin: center center;
+  -webkit-transform: translateX(40px) skewX(7deg);
+  -moz-transform: translateX(40px) skewX(7deg);
+  -ms-transform: translateX(40px) skewX(7deg);
+  -o-transform: translateX(40px) skewX(7deg);
+  transform: translateX(40px) skewX(7deg);
+  -o-transition: -o-transform 0.3s, opacity 0.3s;
+  -ms-transition: -ms-transform 0.3s, opacity 0.3s;
+  -moz-transition: -moz-transform 0.3s, opacity 0.3s;
+  -webkit-transition: -webkit-transform 0.3s, opacity 0.3s;
+}
+
+.wsmenu>.wsmenu-list>li:hover>.wsmegamenu {
+  opacity: 1;
+  visibility: visible;
+  -webkit-transform: rotateX(0deg);
+  -moz-transform: rotateX(0deg);
+  -ms-transform: rotateX(0deg);
+  -o-transform: rotateX(0deg);
+  transform: rotateX(0deg);
+}
```

Comment:
Same thing for css files. Basically all files :)

Reply:
fixed